source/base/mpi.cc

```diff
@@ -183,76 +183,7 @@ namespace Utilities
                  const int        tag,
                  MPI_Comm *       new_comm)
     {
-#  if DEAL_II_MPI_VERSION_GTE(3, 0)
       return MPI_Comm_create_group(comm, group, tag, new_comm);
-#  else
```

Comment:
And this means that we can get rid of this function as well. (Also as a follow-up.)